madhumitha/kra-home-copy.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta name="viewport" content="viewport-fit=cover, width=device-width, initial-scale=1.0">
+
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans&display=swap" rel="stylesheet">
+    <title>Home</title>
+    <link href="reset.css" rel="stylesheet" type="text/css">
+    <link href="kra-home-copy.css" rel="stylesheet" type="text/css">
+</head>
+
+<body>
+    <div class="nav-bar shadow">
+        <div class="logo">
+            <fig><img src="Asset-Logo.svg" alt="zilker"></fig>
+        </div>
+
+        <ul class="links">
+            <li class="nav-item"><a href="#home" class="my-home">MY HOME</a></li>

```

Comment:
Remove the bullet icons which appears on these list items

Reply:
added reset.css.. this removes the bullet on the list items